src/object_store.hpp

```diff
@@ -47,6 +47,10 @@ class ObjectStore {
     // get the last set schema version
     static uint64_t get_schema_version(Group const& group);
 
+    // set schema version
+    // FIXME remove this after integrating OS's migration related logic into Realm java
+    static void set_schema_version(Group& group, uint64_t version);
```

Comment:
Can this throw in some cases? E.g. if the version number if lower than the existing. This should be documented.